spec/models/content_asset_spec.rb

```diff
@@ -1,5 +1,30 @@
 require "rails_helper"
 
 RSpec.describe ContentAsset, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe "with an attached file" do
+    before(:each) do
+      @content_asset = ContentAsset.new(title: "Title")
```

Comment:
I'd like to explicitly set the subject for tests where possible.
https://relishapp.com/rspec/rspec-core/v/3-10/docs/subject/explicit-subject

For this spec it's look something like this

```
subject { ContentAsset.new(title: Faker::Lorem.word) }

before(:each) do
  subject.avatar.attach(io: FILE)
  subject.save!
end

it "has a file attached" do
  expect(subject.avatar).to be_attached
end
```

Reply:
Explicit subjects like this one above are not too hard to read, although I prefer if the code to instantiate it is not too far from where its used, otherwise you're always flipping backwards to see what is being tested.

But let's avoid implicit subjects because its not always obvious how they are instantiated. I think anonymous subjects make tests harder to read.   

Summary: lets use Tony's idea !  But take care not to end up making the tests harder to read.